src/ggrc/converters/handlers/handlers.py

```diff
@@ -273,8 +273,20 @@ def get_value(self):
 class UsersColumnHandler(UserColumnHandler):
   """Handler for multi user fields."""
 
+  def _missed_mandatory_person(self):
+    """Create response for missing mandatory field"""
+    self.add_warning(errors.OWNER_MISSING, column_name=self.display_name)
+    return [get_current_user()]
+
   def parse_item(self):
+    """Parses multi users field."""
     people = set()
+    if self.raw_value in {"-", "--", "---"}:
+      if not self.mandatory:
+        self.set_empty = True
+        return None
+      return self._missed_mandatory_person()
```

Comment:
QQ, is it ok when instance has already been created and when you will setup `--` to mandatory role column, this will lead to remove setup acl and create only one acl instance related to user who initiate that procedure?